test/e2e/performance/README.md

```diff
@@ -0,0 +1,50 @@
+# Kubestellar performance regression testing
+This is a simple performance test that aims to identify major regressions. 
+
+The setup is a single bindingpolicy and a single deployment. The test adds the deployment to the WDS and measures the amount of time it takes to create the WDS object, the Binding object, the ManifestWork object, and the deployment object on the WEC. The starting time is just prior to adding the WDS object and the end time is just after the deployment object is observed on the WEC. The other timings are obtained from the objects creation timestamp. The test intentionally looks at seconds rather than milliseconds since we expect this test to run on VMs and and anything below seconds is likely just noise. In addition, the test uses creation timestamps which have seconds granuality.
```

Comment:
This text is still murky. It starts with one sentence that mentions two things, without ordering. The second sentence mention other things. The third sentence says when the timer starts. I think that a clear description would have one enumeration of actions taken, in order, including the starting of the timer.

Reply:
The latest version (not seen here) is better. It now focuses exclusively on the state changes involved in the timing, and makes no mention of any of the setup, not even the creation of the BindingPolicy. I think it would be even better to be a little explicit about the setup. I will follow up with new review comments.